app/components/Nav/Nav.tsx

```diff
@@ -50,7 +50,7 @@ export const Nav = ({ routes, dropdown = false, appId, ariaLabel }: NavProps) =>
         window.location.href = `/account/${accountId}/${appId}/${mobilePageSelect}`
       }
     }
-  }, [appId, mobilePageSelect])
+  }, [accountId, appId, mobilePageSelect])
```

Comment:
Do we even still need this useEffect?

Reply:
not sure, was just cleaning up linting errors